mlflow/utils/databricks_utils.py

```diff
@@ -31,7 +31,8 @@ def wrapper(*args, **kwargs):
                 if context is not None and hasattr(context, name):
                     return getattr(context, name)
             except Exception:
-                return f(*args, **kwargs)
+                pass
+            return f(*args, **kwargs)
```

Comment:
The original code returns `f(*args, **kwargs)` only when an error occurs when attempting to use the REPL context.